magenta/models/rl_tuner/rl_tuner_ops.py

```diff
@@ -0,0 +1,302 @@
+"""Helper functions to support the RLTuner and NoteRNNLoader classes."""
+
+import os
+import random
+
+from magenta.common import tf_lib
+
+import numpy as np
+import tensorflow as tf
+
+LSTM_STATE_NAME = 'lstm'
+
+# Number of output note classes. This is a property of the dataset.
+NUM_CLASSES = 38
+
+# Default batch size.
+BATCH_SIZE = 128
+
+# Music-related constants.
+INITIAL_MIDI_VALUE = 48
+NUM_SPECIAL_EVENTS = 2
+MIN_NOTE = 48  # Inclusive
+MAX_NOTE = 84  # Exclusive
+TRANSPOSE_TO_KEY = 0  # C Major
+DEFAULT_BPM = 80.0
```

Comment:
for consistency with the rest of our code, should call this DEFAULT_QPM.

More details here: https://groups.google.com/a/tensorflow.org/d/msg/magenta-discuss/b33XYjXjj-U/ozfaXAvwDAAJ